specification/summary-care-record.yaml

```diff
@@ -437,6 +437,8 @@ components:
         The user role ID (URID) for the current session. Also known as a user role profile ID (URPID).
 
         For more details, see [determine the user’s role](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/user-restricted-restful-apis-nhs-identity-combined-authentication-and-authorisation#step-6-determine-the-user-s-role).
+        
+        This field is optional. 
```

Comment:
This field is not optional. SCR API uses it in further communication with Spine.